Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1050,6 +1052,19 @@ inline ScrollSnapStop BuilderConverter::convertScrollSnapStop(BuilderState&, con
     return fromCSSValue<ScrollSnapStop>(value);
 }
 
+inline std::optional<ScrollbarColor> BuilderConverter::convertScrollbarColor(BuilderState& builderState, const CSSValue& value)
+{
+    if (is<CSSPrimitiveValue>(value)) {
+        ASSERT(value.valueID() == CSSValueAuto);
+        return std::nullopt;
+    }
+
+    ASSERT(value.isPair());
+    const auto& pair = downcast<CSSValuePair>(value);
+
+    return std::make_optional<ScrollbarColor>(builderState.colorFromPrimitiveValueWithResolvedCurrentColor(downcast<CSSPrimitiveValue>(pair.first())), builderState.colorFromPrimitiveValueWithResolvedCurrentColor(downcast<CSSPrimitiveValue>(pair.second())));
```

Comment:
We should not resolve currentColor here, we should use `BuilderState::colorFromPrimitiveValue`